WS2012R2/lisa/setupscripts/fcopy_repeated_delete.ps1

```diff
@@ -58,102 +58,19 @@ param([string] $vmName, [string] $hvServer, [string] $testParams)
-
-    .\bin\plink -i ssh\${sshKey} root@${ipv4} "ps -ef | grep '[h]v_fcopy_daemon\|[h]ypervfcopyd' > /tmp/fcopy_present"
-    if (-not $?) {
-        Write-Error -Message  "ERROR: Unable to verify if the fcopy daemon is running" -ErrorAction SilentlyContinue
-        Write-Output "ERROR: Unable to verify if the fcopy daemon is running"
-        return $False
-    }
-
-    .\bin\pscp -i ssh\${sshKey} root@${ipv4}:/tmp/fcopy_present .
-    if (-not $?) {
-        Write-Error -Message "ERROR: Unable to copy the confirmation file from the VM" -ErrorAction SilentlyContinue
-        Write-Output "ERROR: Unable to copy the confirmation file from the VM"
-        return $False
-    }
-
-    # When using grep on the process in file, it will return 1 line if the daemon is running
-    if ((Get-Content $filename  | Measure-Object -Line).Lines -eq  "1" ) {
-        Write-Output "Info: hv_fcopy_daemon process is running."
-        $retValue = $True
-    }
-
-    del $filename
-    return $retValue
-}
-
-#######################################################################
-#
-#   Checks if test file is present
-#
-#######################################################################
-function check_file([String] $testfile)
-{
-    .\bin\plink -i ssh\${sshKey} root@${ipv4} "wc -c < /mnt/$testfile"
-    if (-not $?) {
-        Write-Output "ERROR: Unable to read file /mnt/$testfile." -ErrorAction SilentlyContinue
-        return $False
-    }
-    return $True
-}
-
-#######################################################################
+####################################################################################
 #
-#   Mount disk
+# Checking if the file is present on the guest and file size is matching
 #
-#######################################################################
-function mount_disk()
-{
-    . .\setupScripts\TCUtils.ps1
-
-    $driveName = "/dev/sdb"
-
-    $sts = SendCommandToVM $ipv4 $sshKey "(echo d;echo;echo w)|fdisk ${driveName}"
-    if (-not $sts) {
-        Write-Output "ERROR: Failed to format the disk in the VM $vmName." | Tee-Object -Append -file $summaryLog
-        return $False
-    }
-
-    $sts = SendCommandToVM $ipv4 $sshKey "(echo n;echo p;echo 1;echo;echo;echo w)|fdisk ${driveName}" | Tee-Object -Append -file $summaryLog
-    if (-not $sts) {
-        Write-Output "ERROR: Failed to format the disk in the VM $vmName."
-        return $False
-    }
-
-    $sts = SendCommandToVM $ipv4 $sshKey "mkfs.ext3 ${driveName}1"
-    if (-not $sts) {
-        Write-Output "ERROR: Failed to make file system in the VM $vmName." | Tee-Object -Append -file $summaryLog
-        return $False
-    }
-
-    $sts = SendCommandToVM $ipv4 $sshKey "mount ${driveName}1 /mnt"
-    if (-not $sts) {
-        Write-Output "ERROR: Failed to mount the disk in the VM $vmName." | Tee-Object -Append -file $summaryLog
+####################################################################################
+function check_file_vm(){

```

Comment:
this check_file_vm function should no longer be required, why not just call out directly CheckFile instead?

Reply:
Hi Chirs,
Thank you for your review, will update it to make script more concise. Yes, it can be removed by using checkFile directly, not remove it only because I want to do less change to original script.

Reply:
Done, thank you.